src/backend/sgx/ioctls.rs

```diff
@@ -28,9 +28,15 @@ pub const ENCLAVE_SET_ATTRIBUTE: Ioctl<Write, &SetAttribute<'_>> = unsafe { SGX.
 /// SGX_IOC_VEPC_REMOVE_ALL (0x04) is not used by Enarx.
 
 /// SGX_IOC_ENCLAVE_RESTRICT_PERMISSIONS
-pub const ENCLAVE_RESTRICT_PERMISSIONS: Ioctl<WriteRead, &RestrictPermissions<'_>> =
+pub const ENCLAVE_RESTRICT_PERMISSIONS: Ioctl<WriteRead, &RestrictPermissions> =
     unsafe { SGX.write_read(0x05) };
 
+/// SGX_IOC_ENCLAVE_MODIFY_TYPES
+pub const ENCLAVE_MODIFY_TYPES: Ioctl<WriteRead, &ModifyTypes> = unsafe { SGX.write_read(0x06) };
+
+/// SGX_IOC_ENCLAVE_REMOVE_PAGES
+pub const ENCLAVE_REMOVE_PAGES: Ioctl<WriteRead, &RemovePages> = unsafe { SGX.write_read(0x07) };
+
```

Comment:
The ioctl changes should be one commit. The ioctl additions should be a second commit. Other changes should go in subsequent commits.

Reply:
it's now in one commit.

Reply:
i sliced the whole thing in ttl 8 commits.